ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -507,6 +509,25 @@ KEXT_STATIC int HandleVnodeOperation(
                 {
                     goto CleanupAndReturn;
                 }
+                
+                if (s_osSupportsRenameDetection)
+                {
+                    PrjFSDirectoryXAttrData directoryXattr = {};
+                    SizeOrError xattrResult = Vnode_ReadXattr(currentVnode, PrjFSDirectoryXAttrName, &directoryXattr, sizeof(directoryXattr));
+                    if (xattrResult.error == 0)
```

Comment:
generally we check for ENOATTR to ensure the attribute doesn't exist.  Curious why we check == 0.  Should we be checking against == or != 0 elsewhere?